vendor/bat-native-ledger/include/bat/ledger/ledger_client.h

```diff
@@ -23,12 +24,17 @@ using FetchIconCallback = std::function<void(bool, const std::string&)>;
 
 using LoadURLCallback = std::function<void(const type::UrlResponse&)>;
 
+using LoadURLCallback2 = base::OnceCallback<void(const type::UrlResponse&)>;
```

Comment:
What do you think about naming this just `LoadURLCallback` (without the `2`), so that we have a clean interface for classes implementing `LedgerClient` and new code?

I know that we have a lot of existing uses of the old alias, so we'd need a new name (`LegacyLoadURLCallback` or something) and we'd need to update all of those usage sites.

Reply:
Yeah, that's a good idea!